feature/auth/build.gradle.kts

```diff
@@ -43,5 +43,12 @@ kotlin {
 
             implementation(libs.play.services.auth)
         }
+        // Add Android-specific test dependencies
```

Comment:
We should write unit test for common not only android.
so you don't need to add anything in this gradle in `KMPLibraryConventionPlugin.kt` the kotlin.test is already added